### PR TITLE
QoL changes to improve setup and game installation experience

### PR DESCRIPTION
**Pull request by LostSoulfly**
Small and simple QoL change to pick the (in my experience), most likely installation files.

### CODE REVIEW

**Comment by BradleyShaner** [comment]
Whoops, I didn't realize subsequent changes would be added to this push. 

Sorry if it's a bit messy, I've never worked with xaml before. Basically, I wanted to polish up the setup experience as well as add some QoL fixes.

File: gamevault/Helper/LoginManager.cs [hunk 1]
```diff
@@ -77,6 +77,13 @@ public async Task StartupLogin()
         {
             LoginState state = LoginState.Success;
             if (IsLoggedIn()) return;
+
+            if (Preferences.Get(AppConfigKey.Username, AppFilePath.UserFile).IsNullOrEmpty() && !SettingsViewModel.Instance.ServerUrl.IsNullOrEmpty())
+            {
+                m_LoginState = LoginState.Unauthorized;
+                return;
```
Comment (BradleyShaner) [inline gamevault/Helper/LoginManager.cs:84]: No need to query the server if we know it won't be authorized.

**Comment by Alfagun74** [comment]
Hi there,

Thanks so much for your contribution! I noticed that everything is squashed into a single branch, which makes it a bit challenging to review. Could you please add more comments to your changes to help us understand your reasoning?

Also we usually prefer contributions that reference a ticket on our board.

Thanks again!

**Comment by LostSoulfly** [comment]
Hey, I appreciate the effort you guys are putting into this software. It's great!

I already put all the free time I was willing to contribute into this PR and believed I was descriptive enough about each of my commits. I will opt to maintain my own fork for myself and my friends going forward as long as I use GameVault, so I've closed this PR.

Thanks.

**Comment by Alfagun74** [comment]
I'm really sorry this has been waiting for so long. Having everything in a single PR without references to GitHub issues or at least some comments explaining what you were trying to achieve definitely make it harder to review, especially since we are also actively working on a huge update right now. We will definitely take a closer look and draw some inspiration if we find anything useful, if you don't mind.

**Pull request closed without merging**